typings/index.d.ts

```diff
@@ -538,6 +592,9 @@ declare module 'discord.js' {
     DefaultMessageNotifications: DefaultMessageNotifications[];
     VerificationLevels: VerificationLevel[];
     MembershipStates: 'INVITED' | 'ACCEPTED';
+    ApplicationCommandOptionTypes: ApplicationCommandOptionTypes;
+    InteractionTypes: InteractionTypes;
+    InteractionResponseTypes: InteractionResponseTypes;
```

Comment:
```suggestion
    ApplicationCommandOptionTypes: typeof ApplicationCommandOptionTypes;
    InteractionTypes: typeof InteractionTypes;
    InteractionResponseTypes: typeof InteractionResponseTypes;
```
it would be `typeof x` here right? without `typeof` these would just be typed as numbers when they aren't numbers